toolset/setup/linux/prerequisites.sh

```diff
@@ -62,6 +62,8 @@ RETCODE=$(fw_exists ~/.bash_profile.bak)
 }
 
 sudo sh -c "echo '*               -    nofile          65535' >> /etc/security/limits.conf"
+sudo sh -c "echo '*            hard    rtprio             99' >> /etc/security/limits.conf"
```

Comment:
Would prefer this be done in a separate pull request with a VERY detailed explanation as to why it is needed and what it does.